source/_components/fortigate.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: "Fortigate"
+description: "Instructions on how to integrate Fortigate Firewalls into Home Assistant."
+date: 2019-07-07 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: fortinet.jpg
+ha_category:
+  - Presence Detection
+ha_release: 0.95
+ha_iot_class: Local Polling
+redirect_from:
```

Comment:
```suggestion
redirect_from:
```

Please remove this, this is only needed for older pages

Reply:
fixed in next commit